docs/gadgets/profile/cpu.md

```diff
@@ -135,24 +135,24 @@ Finally, we need to clean up our pod:
 $ kubectl delete pod random
 ```
 
-### With local-gadget
+### With IG
 
 * Generate some kernel load:
 
 ```bash
 $ docker run -d --rm --name random busybox cat /dev/urandom > /dev/null
 ```
 
-* Start local-gadget:
+* Start `ig`:
 
 ```bash
-$ sudo ./local-gadget profile cpu -K --containername random --runtimes docker
+$ sudo ./ig profile cpu -K --containername random --runtimes docker
```

Comment:
For `kubectl-gadget` we are using `kubectl gadget` (so assuming users has it installed in the host). Why we aren't doing the same for `ig`?. If we all agree, it should be done in another PR as it is completed unrelated:

```suggestion
$ sudo ig profile cpu -K --containername random --runtimes docker
```

Reply:
Sometimes we also use `ig profile....` instead of `./ig profile....` in other doc files. We should consolidate the whole documentation (An Issue already exists i think)